### PR TITLE
spell.ignore: add exceptions for this release

### PR DESCRIPTION
**Pull request by apahim**
Adding exceptions to the spell.ignore.

Signed-off-by: Amador Pahim <apahim@redhat.com>

### CODE REVIEW

File: spell.ignore [hunk 1]
```diff
@@ -399,3 +399,12 @@ urlparse
 cpus
 other's
 praveen
+stdin
+null's
+fd
+qemu
+addrs
+hwaddr
+hwaddr
```
Comment (ldoktor) [inline spell.ignore]: You have `hwaddr` twice here...

Comment (apahim) [inline spell.ignore]: fixing

Comment (ldoktor) [inline spell.ignore]: But let me merge it with this slight modification...

Comment (apahim) [inline spell.ignore]: oh, ok.

**Comment by ldoktor** [comment]
Merged with the minor fix https://github.com/avocado-framework/avocado/commit/d4561f6a6f65fab241d40979411909c152263925

**Pull request closed without merging**